ios/RNTPTDocumentView.m

```diff
@@ -2063,6 +2070,78 @@ -(void)selectAnnotation:(NSString *)annotationId pageNumber:(NSInteger)pageNumbe
 }
 
 
+#pragma mark - Set Property for Annotation
+
+- (void)setPropertyForAnnotation:(NSString *)annotationId pageNumber:(NSInteger)pageNumber propertyMap:(NSDictionary *)propertyMap {
+    
+    NSError *error;
+    
+    [self.pdfViewCtrl DocLock:YES withBlock:^(PTPDFDoc * _Nullable doc) {
+        
+        PTAnnot *annot = [self findAnnotWithUniqueID:annotationId onPageNumber:(int)pageNumber];
+        if (![annot IsValid]) {
+            NSLog(@"Failed to find annotation with id \"%@\" on page number %d",
+                  annotationId, (int)pageNumber);
+            annot = nil;
+            return;
+        }
+        
+        [self.toolManager willModifyAnnotation:annot onPageNumber:(int)pageNumber];
+        
+        NSString* annotContents = [RNTPTDocumentView PT_idAsNSString:propertyMap[@"contents"]];
```

Comment:
Are all of these keys defined somewhere? Even just for this file, it would be nicer to declare them as string constants that use raw strings as keys in code.